src/main/java/spoon/support/reflect/code/CtAssertImpl.java

```diff
@@ -46,7 +46,8 @@ public void accept(CtVisitor visitor) {
 	}
 
 	public void setExpression(CtExpression<T> value) {
-		value.setParent(this);
+		if (value != null)
+			value.setParent(this);
```

Comment:
There is an "assertionExpression" and just "expression". Null expression corresponds to the statement `assert cond;`